wcomponents-core/src/main/resources/schema/ui/v1/fieldIndicator.xsd

```diff
@@ -13,7 +13,7 @@
 					</xs:annotation>
 				</xs:element>
 			</xs:sequence>
-			<xs:attribute name="id" type="xs:ID" use="required">
```

Comment:
If the ID field is not required, do they need to be uniquely identified?
As these are nested in the "diagnostable" components, would their ID be the parent component plus an identifier (eg seq)?

Reply:
The is is used for stand-alone WField[Error|Warning]Indicator and not needed for the diagnostic output of Diagnosables. It is a pain but will make sense when we get rid of the XML.